project/frontend/src/containers/CreateCompetition/CreateCompetition.js

```diff
@@ -5,14 +5,14 @@ import LinkButton from "../../components/UI/LinkButton/LinkButton";
 import ParticipantCard from "../../components/ParticipantCard/ParticipantCard";
 import { RiAddCircleLine } from "react-icons/ri";
 import regeneratorRuntime from "regenerator-runtime";
-import { onCompSelect } from "../../containers/SelectCompetition/SelectCompetition.js";
 
 const CreateCompetition = (props) => {
   const [participant, setParticipant] = useState("");
   const [participantList, setParticipantList] = useState([]);
   const [compName, setCompName] = useState("");
   const [startDate, setStartDate] = useState("");
   const [endDate, setEndDate] = useState("");
+  const [dateErrorMessage, setDateErrorMessage] = useState("undefined");
```

Comment:
This should just be an empty string, as there is no error, then we can just check if the string is `""` instead of `"undefined"`

Reply:
I just got a bit nervous setting the starting value to be the same as the valid value, but you think it should be fine ?

Reply:
I think we should assume that there is no error message, and set it to "" by default :)